components/tools/OmeroJava/test/integration/AbstractServerTest.java

```diff
@@ -698,8 +698,8 @@ protected void makeGroupOwner() throws Exception {
      */
     protected omero.client disconnect() throws Exception {
         omero.client oldClient = client;
-        client = null;
         clean();
+        client = null;
```

Comment:
That's a nasty test bug. So is the reason why this wasn't needing in Ice 3.6 that Glacier is doing a better job of cleaning up objects than before?